discojs/discojs-core/src/dataset/data/preprocessing/tabular_preprocessing.ts

```diff
@@ -20,13 +20,13 @@ const sanitize: PreprocessingFunction = {
   apply: (entry: tf.TensorContainer, task: Task): tf.TensorContainer => {
     // if preprocessing a dataset without labels, then the entry is an array of numbers
     if (Array.isArray(entry)) {
-      return entry.map(i => i === undefined ? 0 : i)
+      return entry.map(i => i ?? 0)
     // otherwise it is an object with feature and labels
     } else {
       const { xs, ys } = entry as TabularEntry
       return {
-        xs: xs.map(i => i === undefined ? 0 : i),
-        ys: ys
+        xs: xs.map(i => i ?? 0),
+        ys
```

Comment:
Oh very nice, I'll remember that.
Can you return `ys` directly because it has the same name as the key?